seed/python-sdk/alias/src/seed/core/pagination.py

```diff
@@ -0,0 +1,79 @@
+# This file was auto-generated by Fern from our API Definition.
+
+import asyncio
```

Comment:
can we only include `pagination.py` if there are paginated endpoints?

Reply:
Will add!